packages/connectivity/src/scp-cf/destination/destination-cache.ts

```diff
@@ -211,7 +211,7 @@ async function cacheRetrievedDestination<T extends DestinationCacheInterface>(
  * Sets the custom destination cache instance.
  * Call this method with an instance of {@link DestinationCacheInterface} to override the default cache instance set by the SDK.
  *
- * NOTE: This function should be called at the beginning before any calls to either {@link getDestination} or {@link executeHttpRequest}.
+ * NOTE: This function should be called at the beginning before any calls to either {@link getDestination} or {@link @sap-cloud-sdk/http-client!executeHttpRequest}.
```

Comment:
[q] I notice we use `@link` differently here, could you explain why `{@link executeHttpRequest}` doesn't work?